packages/api/src/modules/providers/subventia/repositories/subventia.repository.ts

```diff
@@ -1,9 +1,21 @@
+import { Siren, Siret } from "dto";
 import MongoRepository from "../../../../shared/MongoRepository";
 import { SubventiaDbo } from "../@types/subventia.entity";
-
 export class SubventiaRepository extends MongoRepository<Omit<SubventiaDbo, "_id">> {
     readonly collectionName = "subventia";
 
+    public async findBySiren(siren: Siren) {
+        return this.collection
+            .find({
+                siret: new RegExp(`^${siren}\\d{5}`),
+            })
```

Comment:
```suggestion
            .find({
                siret: { $regex: new RegExp(`^${siren}\\d{5}`)} ,
            })
```

Les requêtes mongo ont besoin de la commande `$regex` pour prendre en compte les regex. Sinon je pense qu'il va expliciter chercher la regex, pas essayer de l'appliquer

Reply:
Pas sur, il me semble que ça marche comme ça ! Mais ma mémoire est flou !

Reply:
Dans chorus.line la commande $regex n'est pas utilisé. Dans cette situation je pense que les deux marchent bien. l'operateur $regex permets d'utiliser des options en plus pour faire des correspondance par exemple. Si vous avez des préferences. Dites-moi :)

Reply:
Si c'est testé ça me va. J'avais fait un test qui ne marchait pas mais peut-être que je ne faisais pas exactement la même chose